src/safariservices.cs

```diff
@@ -171,6 +176,15 @@ interface SFSafariExtensionHandling
 
 		[Export ("popoverViewController")]
 		SFSafariExtensionViewController PopoverViewController { get; }
+
+		[Mac (10, 12, 4)]
+		[Async (ResultTypeName="SFValidationResult")]
+		[Export ("validateContextMenuItemWithCommand:inPage:userInfo:validationHandler:")]
+		void ValidateContextMenuItem (string command, SFSafariPage page, [NullAllowed] NSDictionary<NSString, NSObject> userInfo, Action<bool, NSString> validationHandler);
```

Comment:
It won't let me do it without the ResultTypeName (or ResultType) on the Async:

error BI1023: bmac: Async method Void ValidateContextMenuItem(System.String, SafariServices.SFSafariPage, Foundation.NSDictionary`2[Foundation.NSString,Foundation.NSObject], SafariServices.SFValidationResult) with more than one result parameter in the callback by neither ResultTypeName or ResultType